src/GeoExt/data/proxy/Protocol.js

```diff
@@ -63,22 +103,28 @@ Ext.define('GeoExt.data.proxy.Protocol', {
      */
     doRequest: function(operation, callback, scope) {
         var me = this,
+            operationParams,
             params,
-            request;
+            request,
+            o,
+            cb,
+            options;
 
         if(GeoExt.isExt5){
-            callback = operation.getCallback();
-            scope = operation.getScope();
-            params = Ext.applyIf(operation.getParams() || {}, me.extraParams || {});
-        }
-        else {
-            params = Ext.applyIf(operation.params || {}, me.extraParams || {});
+            callback = callback || operation.getCallback();
+            scope = scope || operation.getScope();
```

Comment:
I guess this is done because in ExtJS5 there are no arguments ``callback`` and ``scope``?